it/src/main/java/org/corfudb/universe/node/server/process/CorfuProcessManager.java

```diff
@@ -28,10 +29,10 @@
     private final Path serverJar;
     private final Path serverJarRelativePath;
 
-    private final String networkInterface;
+    private final IpAddress networkInterface;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'networkInterface' could be replaced by a local variable.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4653905)